test/Twig/Tests/Node/Expression/CallTest.php

```diff
@@ -91,12 +91,16 @@ public static function customStaticFunction($arg1, $arg2 = 'default', $arg3 = ar
     public function customFunction($arg1, $arg2 = 'default', $arg3 = array())
     {
     }
+
+    private function getArguments($call, $args)
+    {
+        $m = new ReflectionMethod($call, 'getArguments');
+        $m->setAccessible(true);
+
+        return $m->invokeArgs($call, $args);
+    }
 }
 
 class Twig_Tests_Node_Expression_Call extends Twig_Node_Expression_Call
```

Comment:
could this class be removed entirely then and use ``Twig_Node_Expression_Call`` ?